model/src/w3wdatmd.F90

```diff
@@ -416,10 +416,10 @@ SUBROUTINE W3DIMW  ( IMOD, NDSE, NDST, F_ONLY )
     USE W3ODATMD, ONLY: NAPROC, IAPROC
     USE W3SERVMD, ONLY: EXTCDE
     USE CONSTANTS, ONLY : LPDLIB, DAIR
-    USE W3PARALL, ONLY: SET_UP_NSEAL_NSEALM, LSLOC
 #ifdef W3_NL5
     USE W3GDATMD, ONLY: QI5NNZ
 #endif
+    USE W3PARALL, ONLY: SET_UP_NSEAL_NSEALM, LSLOC
```

Comment:
Seems like an unnecessary change.  Reverting would just be one less change in this PR unless there is a specific reason for this change in order

Reply:
done

Reply:
Thanks!

Reply:
@aronroland seems like this update was not pushed to the PR.